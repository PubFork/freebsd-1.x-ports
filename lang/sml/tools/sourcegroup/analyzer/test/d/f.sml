abstraction A = struct val y = B.c end;
