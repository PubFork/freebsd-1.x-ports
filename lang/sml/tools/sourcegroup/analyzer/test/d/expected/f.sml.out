source sml "d/f.sml"
  import structure B 
  export structure A ;

